Sources/xcproj/PBXObject.swift

```diff
@@ -113,6 +113,20 @@ extension Array where Element: Referenceable {
     }
 }
 
+extension Dictionary where Value: Referenceable {
+    public var references: [String] {
+        return self.values.map { $0.reference }
+    }
+
+    public func contains(reference: Key) -> Bool {
+        return self[reference] != nil
+    }
+
+    public func getReference(_ reference: Key) -> Value? {
```

Comment:
We don't need this, do we? If I'm not wrong, we could just access the dictionary directly, couldn't we?

Reply:
I assume this was added for backwards compatibility 👍

Reply:
Yep this was added to minimize updating all callers